src/Entity/Listing/EntityBaseListBuilder.php

```diff
@@ -20,7 +15,7 @@ class EntityBaseListBuilder extends EntityListBuilder {
    */
   public function buildHeader() {
     return [
-      'label' => $this->t('')
+      'label' => '',
```

Comment:
The rule says: a t() call with an empty string is pointless, and well, I kinda agree :) If you want to keep it, we could remove the rule easily